### ISSUE TITLE
Modal screens not working

### ISSUE THREAD

**Issue by objectivehtml** [mattfordham/Reedactor#3]
Haven't anytime to dive into this myself, but was wondering if you have seen this issue. The modals to create link, images, tables, etc, are all not appears. The modal white overlay appears, but no dialog or subsequent modal.

I have been really wanting to dive into Redactor.js but just haven't had time.

**Comment by mattfordham** [comment]
They work for me in Safari. What browser are you using? Do the modals work here?: http://redactorjs.com/

**Comment by objectivehtml** [comment]
Nope, doesn't work in Safari either. I just noticed there is no undo function either. Both the modals and undo work in the Redactor.js site, so it much be something within EE. I have 4 fields setup in my News channel, Redactor being the only FT that is not first party.

**Comment by mattfordham** [comment]
Hmm, weird. Are you using any custom control panel themes?

**Comment by objectivehtml** [comment]
Standard Theme, just did a EE 2.5.2 update, worked great. Then I installed your FT.

**Comment by mattfordham** [comment]
I haven't tested yet in 2.5... I will when I get a chance.
